.gitignore

```diff
@@ -33,3 +33,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+
+.env.local
```

Comment:
This is not required since .env*.local is already added to .gitignore file.

Please remove this change

Reply:
If i do this my .evn.local will be updated to the repo

Reply:
> If i do this my .evn.local will be updated to the repo

no it should not. gitignore already has .env.local ignored

![image](https://github.com/bgwebagency/redesigner.io/assets/13310363/6081cf89-338e-43bd-b309-e5843da5c093)

is it showing the file in your git status?

Reply:
I fixt it!